14-RemoteRepository/README.md

```diff
@@ -0,0 +1,145 @@
+# Working with remote repository
+
+## Introduction
+
+Once we have our remote repository configured we would be able to work with it.
+
+The list of commands below allows to work with remote repository.
+
+### Upstream
+
+Upstream is a branch in central location. If a branch is configured for upstream
+some of the commands (like ```git pull```) will not require the name of remote
+and branch.
+
+Also it is pretty convenient since you may usually see the status of your
+working branch indicated in the command line.
+
+## Commands
+
+### Creating copy of remote repository
+
+* Since we just published our local repository we do not need to create
```

Comment:
What is the purpose of this sentence? Explanation why we do not need to create a local copy is followed by explanation how to do it?